manuscript/draft.Rmd

```diff
@@ -166,3 +177,84 @@ None of the models are true forcasts as they use observed predictor variables (e
 * Need predictions at multiple time scales
 * Prediction quality at short time scales might not be a good indicator of 
   long-term effectiveness
+
+
+
+## Box 1: Ten simple rules for making and evaluating ecological forecasts
```

Comment:
Adding this sounds good, but I'm not sure exactly how to phrase this. Can you propose some language for it?